app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,11 @@
+class OrdersController < ApplicationController
+  def index
+    @orders = current_account.orders
```

Comment:
@orders có bao giờ nil ko e?

vs cả như a Hùng comment, ko cần check có order hay ko để return nhé, ở ngoài view e chỉ cần hiển thị "You have no orders" nếu ko có order nào.